library/src/main/java/com/pokegoapi/api/inventory/ItemBag.java

```diff
@@ -72,7 +72,7 @@ public void addItem(Item item) {
 	 * @throws LoginFailedException  the login failed exception
 	 */
 	public Result removeItem(ItemId id, int quantity) throws RemoteServerException, LoginFailedException {
-		Item item = getItem(id);
+		final Item item = getItem(id);
```

Comment:
no, the scope it's to remove all the sync requests and remove the method on requesthandler